app.py

```diff
@@ -22,22 +22,29 @@ def validate(params):
     if app_id and token and team and channel and user and subtype:
         return True
     else:
+        if not app_id:
+            print("APP_ID is not right!")
+        if not token:
+            print("TOKEN is not right!")
+        if not team:
+            print("TEAM_ID is not right!")
+        if not channel:
+            print("USLACKBOT channel is not right!")
+        print("\n\n\n")
```

Comment:
More pythonic ways for this section can be implemented
https://stackoverflow.com/questions/6581190/check-if-one-of-variables-is-set-to-none

Although we won't get which variable was wrong

Reply:
No so like When I entered TEAM_ID wrong the validate params function was not working
I found it saying this variable to be false